addons/website/static/tests/tours/website_signup_form.js

```diff
@@ -0,0 +1,122 @@
+import { clickOnSave, registerWebsitePreviewTour } from "@website/js/tours/tour_utils";
+
+function fillInputField(selector, value) {
+    return [
+        {
+            content: `Fill input field with value ${value}`,
+            trigger: `${selector}`,
+            run: `edit ${value}`,
+        },
+    ];
+}
+function checkField(value) {
+    return [
+        {
+            content: `Verify if the field is correctly set to '${value}'`,
+            trigger: `input[value='${value}']`,
+            run: () => {}, // This is a check
+        },
+    ];
+}
+
+registerWebsitePreviewTour(
+    "website_signup_form_add_custom_field",
+    {
+        url: "/web/signup",
+        edition: true,
+    },
+    () => [
+        {
+            content: "Select the confirm password field to proceed",
+            trigger: ":iframe input#confirm_password",
+            run: "click",
+        },
+        {
+            content: "Click the `Field` button to add a new field",
+            trigger: ".o_we_bg_brand_primary",
+            run: "click",
+        },
+        {
+            content: "Open the field type dropdown",
+            trigger: "we-select[data-name='type_opt']",
+            run: "click",
+        },
+        {
+            content: "Select the company_name from existing field",
+            trigger: "we-button[data-existing-field='company_name']",
+            run: "click",
+        },
+        {
+            content: "Click the `Field` button to add a new field",
+            trigger: ".o_we_bg_brand_primary",
+            run: "click",
+        },
+        {
+            content: "Open the field type dropdown",
+            trigger: "we-select[data-name='type_opt']",
+            run: "click",
+        },
+        {
+            content: "Select the city from existing field",
+            trigger: "we-button[data-existing-field='city']",
+            run: "click",
+        },
+        {
+            content: "Click the `Field` button to add a new field",
+            trigger: ".o_we_bg_brand_primary",
+            run: "click",
+        },
+        {
+            content: "Set the label of the new field to 'field_1'",
+            trigger: "we-input[data-set-label-text] input",
+            run: "edit field_1",
+        },
+        ...clickOnSave(),
+        {
+            content: "click the profile button",
```

Comment:
```suggestion
            content: "Click the profile button",
```

Check all other places for capitalization.